packages/dashboard/src/features/dashboard/components/connect/CLISection.tsx

```
@@ -1,17 +1,26 @@
-import { CopyButton, cn } from '@insforge/ui';
+import { useState } from 'react';
+import { ChevronDown } from 'lucide-react';
+import { CopyButton, DropdownMenu, DropdownMenuContent, DropdownMenuItem, DropdownMenuTrigger } from '@insforge/ui';
```

ESLint

[error] Replace `·CopyButton,·DropdownMenu,·DropdownMenuContent,·DropdownMenuItem,·DropdownMenuTrigger·` with `⏎··CopyButton,⏎··DropdownMenu,⏎··DropdownMenuContent,⏎··DropdownMenuItem,⏎··DropdownMenuTrigger,⏎`

(prettier/prettier)